demisto_sdk/commands/common/clients/__init__.py

```diff
@@ -45,6 +47,9 @@ def get_client_from_config(
     api_key = client_config.api_key
     auth_id = client_config.auth_id
 
+    if verify_ssl is None:
```

Comment:
this because `demisto-py` behind the scenes check if this value is `None` and if it is it will take the value from the `DEMISTO_VERIFY_SSL` env var.

now consider a situation where this env var is set to true, but we send verify_ssl=false (by default), it would use verify_ssl=false instead true